src/features/kanban/hooks/useKanbanDragDrop.ts

```
@@ -0,0 +1,270 @@
+            if (a.id === activeId) return newIndex - b.columnOrder + 0.5;
+            if (b.id === activeId) return a.columnOrder - newIndex - 0.5;
+            return a.columnOrder - b.columnOrder;
+          });
+
+        // Ensure moved card is at newIndex
+        const withoutActive = destAll.filter((t) => t.id !== activeId);
+        const activeItem = destAll.find((t) => t.id === activeId)!;
+        withoutActive.splice(newIndex, 0, activeItem);
+
+        const orderMap = new Map<string, number>();
+        withoutActive.forEach((t, i) => orderMap.set(t.id, i));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)